.clj-kondo/config.edn

```diff
@@ -6,7 +6,7 @@
     (clojure.core.logic/fresh)
     (clojure.core.logic/matcha)
     (clojure.core.logic/run)
-    (clojure.test/is [query= sql=])
+    (clojure.test/is [query= re= sql=])
```

Comment:
This is in the Kondo config for the Hawk test runner so you don't need to have it here too

Reply:
Done.